BUILD.md

```diff
@@ -347,17 +347,13 @@ Setup Homebrew and components
 
 ### Clone the Repository
 
-Clone the Vulkan-LoaderAndValidationLayers repository:
+Clone the Vulkan-Loader repository:
 
-    git clone https://github.com/KhronosGroup/Vulkan-LoaderAndValidationLayers.git
+    git clone https://github.com/KhronosGroup/Vulkan-Loader.git
```

Comment:
This line probably isn't a good idea. Cloning like this won't get the submodule unless you initialize it afterwards. But given that neither the windows nor linux build instructions include this step, I'd remove it entirely. There's already a section at the top of this file on how to do it right (which applies to all platforms) so I'd remove this and just rely on the sections that goes over cloning.